packages/react-components/src/components/MicrophoneButton.tsx

```diff
@@ -68,6 +163,46 @@ export const MicrophoneButton = (props: MicrophoneButtonProps): JSX.Element => {
       onRenderOffIcon={props.onRenderOffIcon ?? onRenderMicOffIcon}
       strings={strings}
       labelKey={props.labelKey ?? 'microphoneButtonLabel'}
+      menuProps={props.menuProps ?? generateDefaultDeviceMenuPropsTrampoline(props, strings)}
+      menuIconProps={!showDeviceSelectionMenuTrampoline(props) ? { hidden: true } : undefined}
+      split={showDeviceSelectionMenuTrampoline(props)}
```

Comment:
might be breaking so maybe we can't, but should we `Omit` the likes of `split`, `menuIconProps`, `menuProps` from the button interface?